upload/admin/view/template/module/smaily_for_opencart.tpl

```diff
@@ -275,10 +272,9 @@
                     ];
                     // Add options for select.
                     foreach ($cart_options as $value => $name) {
-                      $selected = is_array($abandoned_additional) && in_array($value, $abandoned_additional) ? 'selected' : '';
-                      echo("<option value='$value' $selected>$name</option>");
-                    }
-                    ?>
+                      $selected = is_array($abandoned_additional) && in_array($value, $abandoned_additional) ? 'selected' : ''; ?>
+                      <option value="<?php echo $value; ?>"<?php echo $selected; ?>><?php echo $name; ?></option>
```

Comment:
Missing whitespace between `value` and `selected` attributes.

Reply:
This is fixed